src/util/webauthn.js

```diff
@@ -0,0 +1,64 @@
+/* jshint es5:true */
```

Comment:
Cool - in that case:
- Copy it back over to the vendors directory (it's a thirdparty file)
- Add a link to the source in a comment at the top of the file, as well as the license information (that it's MIT)
- Add comments to any pieces of code that you changed calling out the fact that we changed it and what we changed